### DISCUSSION TITLE
How can nrich-registry be used with custom annotations for validations

### DISCUSSION THREAD

**Discussion by Waterhoax** [croz-ltd/nrich#150] [Q&A] [answered]
Is it somehow possible to use custom annotations for validations on registry entities? Ive found that the [validateEntityData](https://github.com/croz-ltd/nrich/blob/cf332075d8584af830e6f12501862227f552a3b8/nrich-registry/src/main/java/net/croz/nrich/registry/data/controller/RegistryDataController.java#L87) catches constraints such as `@Size` but ignores custom validations.

**Comment by agrancaric** [comment, marked as answer]
That sounds strange, it should work, maybe there is an issue. Could you provide an example?
